.circleci/config.yml

```diff
@@ -99,7 +99,7 @@ jobs:
           command: yarn storybook:build
       - run:
           name: Chromatic
-          command: yarn chromatic
+          command: yarn chromatic --project-token=${CHROMATIC_PROJECT_TOKEN} --exit-zero-on-changes --storybook-build-dir ./dist-storybook
```

Comment:
Something is odd about the storybook of this PR, it seems like it's for V3 for some reason https://5fd1dda724cc620021ace8c5-xqpqgpuszm.chromatic.com/?path=/story/documentation-general--page

Reply:
Also vercel is failing because it can't find `website` package 🤔

Reply:
Ah ok, why do you want to merge this to v3 branch?

Reply:
I don't actually want to merge, just have this branch to release from. The package was deleted from v4.